lambdas/format_input.py

```diff
@@ -7,7 +7,7 @@
 logger = logging.getLogger(__name__)
 
 
-def lambda_handler(event: dict, context: dict) -> dict:  # noqa
+def lambda_handler(event: dict, _context: dict) -> dict:
```

Comment:
Was the underscore to address a linting error or preference? 🤔

Reply:
Yes: because `context` is not used, it passes linting that way.